compiler/rustc_span/src/hygiene.rs

```diff
@@ -350,14 +352,6 @@ impl ExpnId {
     }
 }
 
-/// This struct is meant to be a surrogate for the actual `DepNode` in rustc_middle.
-/// Both types should be kept in sync.
-#[derive(Copy, Clone, Debug, Hash, PartialEq, Eq)]
-pub struct FakeDepNode {
-    pub kind: u16,
-    pub hash: PackedFingerprint,
-}
```

Comment:
There are some back and forth changes in the commits, probably makes sense to cleanup the history.